src/librustc_mir/borrow_check/error_reporting.rs

```diff
@@ -18,6 +18,7 @@ use rustc::mir::{
     VarBindingForm,
 };
 use rustc::ty;
+use rustc::hir;
```

Comment:
it'd be nice to have some tests showing the effect of this commit (as part of the commit, I mean)

Reply:
This is an older commit that I didn't update the tests for when I was working on it, I just used it as a stepping stone to the final error messages. I could edit that commit to add tests for what was being output at the time.

Reply:
not necessary